clearpath_sensors/config/garmin_18x.yaml

```diff
@@ -0,0 +1,6 @@
+nmea_navsat_driver:
+  ros__parameters:
+    port: "/dev/ACM0"
```

Comment:
Should be `/dev/clearpath/gps`

Reply:
Do we need to add a udev rule for this?

Reply:
It is this rule: https://github.com/clearpathrobotics/clearpath_robot/blob/main/clearpath_robot/debian/udev#L7